linux_os/guide/system/software/integrity/fips/fips_crypto_subpolicy/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+title: 'FIPS Must Use a Supported Subpolicy'
+
+description: |-
+    Sub-policies can be used to modify existing crypto policies.
+    Some sub-policies such as <tt>NO-ENFORCE-EMS</tt> reduce the security of the system and should not be used.
+    Other such as <tt>AD-SUPPORT</tt> should only be enabled if operationally required.
```

Comment:
The description should be aligned with the OVAL check. I think you should a list of specific subpolicies that are allowed by the OVAL check. That way the people reading the descirption will why they fail the rule.